lib/controller.rb

```diff
@@ -6,6 +6,18 @@
 
 class WeekThreeAssessment < Sinatra::Base
 
-  # Your code will go here.
+  get '/' do
+    erb :index
+  end
 
+  def user
+    User.new
+  end
+
+  get '/name/:name' do
+    if user.valid?
+      user.save
+    end
+  end
```

Comment:
The route is set up correctly but you are not performing the right things inside the block. Create a User with `name` attribute set to hold a value of "Amber"  and pass it in to the view using an instance variable .